www/src/pages/guidelines/typography.js

```diff
@@ -0,0 +1,600 @@
+          <Button
+            bg="black"
+            mx="auto"
+            mb={3}
+            css={{ color: `black !important` }}
+          >
+            Start a free trial
+          </Button>
+          <Text color="blackFade.60" fontSize={1}>
+            14 day free trial — no credit card required
+          </Text>
+
+          <Box display={{ lg: `flex ` }} mt={9} textAlign="left">
+            <MarketingColumn title="Scale to the entire internet">
+              Forget complicated deploys with databases and servers and their
+              expensive, time-consuming setup costs, maintenance, and scaling
+              fears — Gatsby builds your site as “static” files which can be
+              deployed easily on various services.
+            </MarketingColumn>
+            <MarketingColumn title="Bring your own data">
+              Gatsby’s rich data plugin ecosystem lets you build sites with the
+              data you want — pull data from headless CMSs, SaaS services, APIs,
+              databases, your file system, and more.
+            </MarketingColumn>
+          </Box>
+          <Box display={{ lg: `flex ` }} mt={7} pb={8} textAlign="left">
+            <MarketingColumn title="Speed past the competition">
+              Gatsby.js builds the fastest possible website. Instead of waiting
+              to generate pages when requested, pre-build pages and lift them
+              into a global cloud of servers — ready to be delivered instantly
+              to your users wherever they are.
+            </MarketingColumn>
+            <MarketingColumn title="Static Progressive Web Apps">
+              Gatsby.js is a static PWA (Progressive Web App) generator. You get
+              code and data splitting out-of-the-box. Gatsby loads only the
+              critical HTML, CSS, data, and JavaScript so your site loads as
+              fast as possible. Once loaded, Gatsby prefetches resources for
+              other pages so clicking around the site feels incredibly fast.
+            </MarketingColumn>
+          </Box>
+        </Box>
+      </Container>
+    </ThemeProvider>
+
+    <Container>
+      <Box bt={1} py={8} my={8}>
+        <Text color="grey.60" fontSize={1} textAlign="center">
+          Cards
+        </Text>
+        <Flex mt={7} flexWrap="wrap" justifyContent="space-around">
+          <Starter mx={6} />
+          <Blog mx={6} />
+        </Flex>
+      </Box>
+    </Container>
+
+    <Container>
+      <Text color="grey.60" fontSize={1} textAlign="center">
+        Long-form text (with sidebar on large screens)
+      </Text>
+      <Flex py={4} my={4}>
+        <Box mt={8} mr="auto" pr={7} display={{ xxs: `none`, md: `block` }}>
+          <Box
+            css={{
+              borderRightWidth: 1,
+              borderRightStyle: `solid`,
+              borderColor: `#F5F5F5`,
+            }}
+            pr={7}
+          >
+            <Heading fontWeight="1" fontSize={3} mb={5}>
```

Comment:
This should probably be an h2 or h3 based on the page hierarchy.

Reply:
I changed the container tag from `div` to `section` in fbca644 to be (hopefully) able to use `<h1>`.